components/manager/scripts/validate.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
If we'll want to allow bootstrapping outside of the CLI, we'll need to make several adjustments to the code as it is.

the relevant question is whether the user/pass are needed as part of the workflow code or as part of the CLI code that runs during bootstrap. at the moment, it's the latter (or possibly, both). therefore, the validation should be as part of the CLI, so in case a user uses a different workflow, the validation will still happen.
in theory, if we allow bootstrapping via other means than the CLI, then the validation will *also* need to happen as part of the workflow - but not instead of the one in the CLI.
at the moment, i think the best is to go for a single validation in the CLI, and not do a double validation for something that might or might not happen in the future.